authors/apps/articles/exceptions.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.exceptions import APIException
+
+
+class NotFound(APIException):
+    """
+    Provide default 404 status code
+    """
+    status_code = 404
+
+
+exception_message = {
+    'article_does_not_exist': "Article does not exist. "
+    "Check provided Article slug."
+}
+
+
+class IncorrectValues(APIException):
```

Comment:
How is this exception different from `Not Found` exception on line 4.

Reply:
Worked on it